app-utils/ouch/autobuild/defines

```diff
@@ -0,0 +1,13 @@
+PKGNAME="ouch"
+PKGDES="Painless compression and decompression for your terminal"
```

Comment:
A command line (de)compression utility with support for multiple archive formats